app/src/test/java/org/mozilla/focus/utils/UrlUtilsTest.kt

```diff
@@ -186,4 +186,27 @@ class UrlUtilsTest {
         assertEquals("random.mozilla.org", UrlUtils.stripCommonSubdomains("random.mozilla.org"))
         assertEquals(null, UrlUtils.stripCommonSubdomains(null))
     }
+
+    @Test
+    fun testToDisplayUrl() {
```

Comment:
I wonder if we should be more expressive with our test names (and testing style), even if it's inconsistent with the rest of the file. I think the win for readability when tests fail is worth it.